app/graphql/types/query_type.rb

```diff
@@ -31,4 +31,20 @@ def events(after: nil, type: nil)
     events = events.with_source(type) if type
     events
   end
+
+  field :employees, [Types::EmployeesType] do
+    description "Employee events from bambooHR"
+    argument :type, String, required: false
```

Comment:
It's probably worth defining an [enum](https://graphql-ruby.org/type_definitions/enums) for this argument instead of accepting arbitrary strings.